src/time_evolution/mesolve.jl

```diff
@@ -141,6 +153,7 @@ where
 
 - The states will be saved depend on the keyword argument `saveat` in `kwargs`.
 - If `e_ops` is empty, the default value of `saveat=tlist` (saving the states corresponding to `tlist`), otherwise, `saveat=[tlist[end]]` (only save the final state). You can also specify `e_ops` and `saveat` separately.
+- If `H` is an [`OperatorQuantumObject`](@ref), `ψ0` is a [`KetQuantumObject`](@ref) and `c_ops` is `Nothing`, the function will call [`sesolveProblem`](@ref) instead.
```

Comment:
same here

btw, I think it should be `sesolve` here instead of `sesolveProblem`

```suggestion
- If `H` is an [`Operator`](@ref), `ψ0` is a [`Ket`](@ref) and `c_ops` is `Nothing`, the function will call [`sesolve`](@ref) instead.
```